templates/admin/library_report_detail.html

```diff
@@ -0,0 +1,202 @@
+{% load static humanize %}
+<!DOCTYPE html>
+<html>
+  <head>
+    <script defer
+            data-domain="preview.boost.org"
```

Comment:
why "preview.boost.org" ?

Reply:
That was copied from the base.html file but I couldn't `extend` it because the body class needed to be different.  It looks like that script is for analytics which I don't think needs to be in the admin page anyway so I'm going to just remove it.